Source/WebInspectorUI/UserInterface/Views/MediaDetailsSidebarPanel.css

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
This file needs a license header as well. The other CSS files will have one formatted for CSS (since `//` isn't valid in CSS).